image_generation/lcm_dreamshaper_v7/cpp/README.md

```diff
@@ -31,9 +51,7 @@ conda env config vars set LD_LIBRARY_PATH=$CONDA_PREFIX/lib:$LD_LIBRARY_PATH
 
     ```shell
     git submodule update --init
-    conda activate openvino_lcm_cpp
     python -m pip install -r requirements.txt
-    python -m pip install ../../../thirdparty/openvino_tokenizers/[transformers]
```

Comment:
Why is this line removed?

`optimum-cli` needs `openvino_tokenizers`. I know that `requirements.txt` install `openvino_tokenizers`, but it may provide an outdated version compared to our submodule.